src/tests/quantities.rs

```diff
@@ -92,6 +92,15 @@ storage_types! {
             &<degree_fahrenheit as Conversion<V>>::constant(ConstantOp::Add).value());
     }
 
+    #[test]
+    fn all_quantity_types_are_unpin() {
+        fn assert_unpin<T: Unpin>() {}
+
+        assert_unpin::<Length>();
+        assert_unpin::<Mass>();
+        assert_unpin::<ThermodynamicTemperature>();
+    }
+
```

Comment:
Move this check into src/tests/asserts.rs. We can have checks for each storage type that is supported by `uom`:

```rust
    assert_impl_all!(qa_impl; QuantityArguments<Q<Z0, Z0, Z0>, U<V>, V, meter>,
        Clone, Copy, Debug, Display, LowerExp, Unpin, UpperExp);
```